airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -288,6 +288,26 @@
    */
   Optional<Map<String, String>> getDiscoverJobKubeNodeSelectors();
 
+  /**
+   * Define one or more Job pod annotations. Each kv-pair is separated by a `,`.
```

Comment:
Can you include in the comment that this annotation is used as a fallback in case other annotations aren't set, as well as, used as the sync job pod annotation.